.github/ci_commit_pins/xla.txt

```diff
@@ -1 +1 @@
-10db402a64309daede957462c66d5664de182d48
+3ab92edcdd01c891fa78bab6335898d3d16b8005
```

Comment:
Did you mean to change this?

Reply:
yes, there's a corresponding PR on the XLA side.